### PR TITLE
refactor version to a package

### PR DESCRIPTION
**Pull request by rphillips**
* So we can include the version within the handshake

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ script:
   - gox
     -osarch "linux/386 linux/amd64 darwin/amd64"
     -output="dist/{{.Dir}}_{{.OS}}_{{.Arch}}"
-    -ldflags="-X main.Version=$(git symbolic-ref -q --short HEAD || git describe --tags --exact-match)"
+    -ldflags="-X github.com/racker/rackspace-monitoring-poller/version.Version=$(git symbolic-ref -q --short HEAD || git describe --tags --exact-match)"
```
Comment (itzg) [inline .travis.yml]: Ah, very cool

File: config/config_test.go [hunk 2]
```diff
@@ -265,7 +265,7 @@ func TestConfig_ParseFields(t *testing.T) {
 			fields: getConfigFields(),
 			args: []string{
 				"monitoring_endpoints",
-				"127.0.0.1,0.0.0.0",
+				"127.dev,0.0.0.0",
```
Comment (itzg) [inline config/config_test.go]: What was this change for?

Comment (rphillips) [inline config/config_test.go]: search and replace issue

Comment (rphillips) [inline config/config_test.go]: fixed

**Pull request merged**